src/components/CioQuiz/actions.ts

```diff
@@ -1,4 +1,5 @@
-import { NextQuestionResponse, QuizResultsResponse } from '../../types';
+// eslint-disable-next-line import/no-cycle
```

Comment:
It wouldn't cause a problem with types but it's just ugly. To make it go away we would have to move a lot of types around which I felt it would get uglier so I just silenced it. But I can give it another stab.

Reply:
I'm assuming you gave it another go and it's still ugly. @deomsj any concern with this?

Reply:
tech debt worth cleaning fo sho. 

I expect that this could only cause issues at build time, not during run time. so.... as long as we are happy with our test coverage, then this issue can live another day and be tackled as a separate tech debt ticket focused on resolving circular-imports